assets/src/edit-story/app/media/utils/useExistingData.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useCallback } from 'react';
+/**
+ * Internal dependencies
+ */
+import useStory from '../../story/useStory';
+
+function useExistingData({ updateMedia }) {
```

Comment:
`useExistingData` is not really very specific. We should come up with a better name.

That said, it makes more sense to just merge the two `useProcess*` hooks into one, and removing this hook here again.

Reply:
Merged into `useProcessMedia`